src/Localization/Localization/test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerFactoryTest.cs

```diff
@@ -1,6 +1,7 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable
```

Comment:
You're using nullable types. Why is this needed?

Reply:
Some of these tests have dozens of null errors. I would like for new tests to be authored "correctly", but fixing these tests to either be correct or just litter it with null-forgiveness seems like a massive chore with low payout.